keywording/concepts/text.xml

```diff
@@ -19,7 +19,7 @@ each given arch (<c>sparc</c>, <c>ppc</c>, <c>x64-macos</c>, ...).
 
 <note>
 The term 'arch' is used in this sense for historical reasons. As a result
-of <uri link="https://www.gentoo.org/glep/glep-0022.html">GLEP 22</uri>
+of <uri link="https://www.gentoo.org/glep/glep-0053.html">GLEP 53</uri>
```

Comment:
Historically, "arch" is inaccurate ever since GLEP 22. So I'd rather keep the old reference here.

Reply:
Makes sense in the context, I agree. Reverted. Would you like a note saying that GLEP 22 has since been replaced by GLEP 53?